crates/fluvio/src/consumer/retry.rs

```diff
@@ -47,18 +101,27 @@ enum ConsumerRetryState {
     Terminated,
 }
 
-/// A consumer stream that automatically retries on failure.
-///
-/// In this refactored version we remove the mutex by taking ownership of
-/// the consumer stream whenever we start a new retry task. When the task finishes,
-/// the stream is returned.
-pub struct ConsumerRetryStream {
-    inner: ConsumerRetryInner,
-    state: ConsumerRetryState,
-    /// The consumer stream is stored directly (inside an Option for ownership transfer).
-    stream: Arc<Mutex<BoxConsumerStream>>,
-    strategy: Arc<dyn ReconnectStrategy>,
-    notify: Arc<Notify>,
+/// A command to be sent to the retry consumer stream.
+enum TaskCommand {
+    Commit,
+    Flush,
+}
+
+impl TaskCommand {
+    async fn apply(self, stream: &mut BoxConsumerStream) {
+        match self {
+            TaskCommand::Commit => {
+                if let Err(e) = stream.offset_commit().await {
+                    warn!(target: SPAN_RETRY, "Commit failed: {}", e);
+                }
+            }
+            TaskCommand::Flush => {
+                if let Err(e) = stream.offset_flush().await {
```

Comment:
same comment.  error must not be ignored

Reply:
fixed